matrix/matrix_class.py

```diff
@@ -351,6 +365,38 @@ def __pow__(self, other):
     def dot_product(cls, row, column):
         return sum(row[i] * column[i] for i in range(len(row)))
 
+    @classmethod
+    def cross_product(cls, first: "Matrix", other: "Matrix") -> "Matrix":

```

Comment:
Thank you for writing this method but a better approach would be to just use a method like so:
```python
def cross_product(self, other: "Matrix") -> "Matrix":
    ...
```
This would allow us to do something like this:
```python
matrix1.cross_product(matrix2)
```

Reply:
Just finished doing that.